modules/installation/pages/install-proxy.adoc

```diff
@@ -1,8 +1,130 @@
 
 
 
+[[installation-proxy-requirements]]
+== SLES KVM Requirements
 
-Watch this space ...
+Enter the following settings when creating a new virtual machine using [command]``virt-manager`` (replace [literal]``<version>`` with the actual version string):
+
+[cols="1,1", options="header"]
+|===
+| KVM Settings for SLES
+| Installation Method: | Local install media (ISO image or CDROM)
+| OS:                  | Linux
+| Version:             |``SLE-``[literal]``<version>````-Server-x86_64-GM-DVD1.iso``
+| Memory:              | _Test Server_ Minumum 2{nbsp}GB
+|                      | _Production Server_ Minimum 8{nbsp}GB
+| CPUs:                | 2
+| Storage Format:      | ISO 3{nbsp}GB
+| Disk Space:          | 230{nbsp}GB split between
+|                      | [path]``/`` _(root)_ Minimum 24{nbsp}GB
+|                      | (Virtual Disk 1) [path]``/srv`` Minimum 100{nbsp}GB
+|                      | (Virtual Disk 2) [path]``/var/cache`` _(Squid)_ Minimum 100{nbsp}GB
+| Name:                | example-proxy
+| Network              | Bridge br0
+|===
+
+
+
+[[installation-proxy-sles-settings]]
+=== SLES KVM Settings
+
+This section provides guidance on installation of {productname} Proxy
+utilizing the full installation media with KVM and [command]``virt-manager``.
+This section assumes you have previously setup an account with SCC and downloaded the SLES full installation media.
+
+
+
+.Procedure: Preparing for SLES Installation
+. In [command]``virt-manager`` select menu:File[New Virtual Machine].
+
+. Select btn:[Local install media (ISO image or CDROM)].
+
+. Ensure btn:[Use ISO Image] is selected.
+Then click btn:[Browse] and locate the full SLES image you downloaded from your SCC account.
+
+. Configure your machine with at least 2 GB RAM and a minimum of 2 CPUs.
+
+. Create a storage device with a minimum of 230 GB storage space for the installation.
+During the partitioning setup of the SLES installation this disk should be partitioned into the following disks:
++
+// FIXME: not sure about swap space
+// |4{nbsp}GB Swap space
++
+[cols="1", options="header"]
+|===
+| Disk Space Requirements
+| 100{nbsp}GB XFS partition (or dedicated virtual disk) for [path]``/srv/``
+| 100{nbsp}GB XFS partition (or dedicated virtual disk) for [path]``/var/cache/``
+|===
+
+. The remaining storage space will be used by the operating system for the root partition.
+Select btn:[Finish] to begin the installation.
+
+Installation of {sles} will begin.
+For more information on completing an installation of {sles}, see: https://documentation.suse.com/sles/15-SP1/html/SLES-all/art-sle-installquick.html.
+
+
+
+[[installation-proxy-prep]]
+== Prepare SLES for SUSE Manager Proxy
+
+[[proc.jeos.susemgr.prep]]
+.Procedure: Preparing SLES for {productname} Proxy Installation
+
+// Most steps are currently needed because of 4.0 workarounds
+. Log in as `root`.
+
+. Uninstall the `sles-release` package:
++
+
+----
+rpm -e --nodeps sles-release
+----
```

Comment:
Ah, so this is again this special case where the user installs a SLES and converts it into a SUMA proxy. If this is intended, it is ok for me. Of course one could also simply boot from SLE-15-SP1-Installer-DVD-x86_64-GM-DVD1.iso and install the SUMA Proxy product from the very beginning. But I'm sure you are aware of that.